src/msw/utils.cpp

```diff
@@ -550,6 +553,15 @@ bool wxUnsetEnv(const wxString& variable)
     return wxDoSetEnv(variable, NULL);
 }
 
+// ----------------------------------------------------------------------------
+// security
+// ----------------------------------------------------------------------------
+
+void wxSecureZeroMemory(void* p, size_t n)
+{
+    RtlSecureZeroMemory(p, n);
```

Comment:
Unfortunately this results in a build failure when using MinGW (not -w64). I don't know if we can just declare this function ourselves or if we need to avoid using it with plain old MinGW.

Reply:
It appears that `wxSecretValue::Wipe`'s MSW implementation calls `::SecureZeroMemory`, which at least on MSVS is just a plain `#define` for `RtlSecureZeroMemory`. Since presumably `wxSecretStore` already compiles with MinGW, is `SecureZeroMemory` something different on MinGW in that case (and is using it the solution here)?

As may be self-evident, I don't have MinGW installed myself.

Reply:
The MinGW issue is now resolved by using the same generic implementation from *nix implementation in case of `__MINGW32__`.